tomcat/src/test/java/com/linecorp/armeria/server/tomcat/TomcatServiceTest.java

```diff
@@ -110,6 +112,7 @@ public void jarBasedWebAppWithAlternativeRoot() throws Exception {
                         .startsWith("application/java");
                 assertThat(res.getFirstHeader(HttpHeaderNames.CONTENT_LENGTH.toString()).getValue())
                         .isEqualTo("1361");
+                EntityUtils.consume(res.getEntity());
```

Comment:
Without this, Apache HC will close the connection before reading the response fully.